sdcm/sct_config.py

```diff
@@ -1307,7 +1307,13 @@ def _check_backend_defaults(backend, required_params):
                 for ami_id, region_name in zip(ami_list, region_names):
                     tags = get_ami_tags(ami_id, region_name)
                     assert 'user_data_format_version' in tags.keys(), \
-                        f"\n\t'user_data_format_version' tag missing from [{ami_id}] on {region_name}\n\texisting tags: {tags}"
+                        f"\n\t'user_data_format_version' tag missing from [{ami_id}] on {region_name}\n\texisting " \
+                        f"tags: {tags}"
+
+        url_variable_names = ['new_scylla_repo', 'scylla_repo_m', 'scylla_repo_loader', 'scylla_mgmt_repo',
+                              'scylla_mgmt_agent_repo', 'scylla_mgmt_agent_repo']
+        extract_all_scylla_urls_from_file(urls=[self.get(
+            url_variable_name) for url_variable_name in url_variable_names if self.get(url_variable_name)])
```

Comment:
@bentsi 
Thanks for your comments
1. The function `extract_all_scylla_urls_from_file` returns list of scylla_version and also validate each URL.
For this case, the result is unnecessary.

2.The main goal is not to affect performance, as much as possible, when extracting URLs from the file.
Sometimes a request for a request can take a long time, so there is one function that returns and checks every link found.
I can separate it into some useless functions (because I only used them to create one function).

3. I fixed the logic of those lines